src/test/java/com/hivemq/cli/DefaultCLIPropertiesTest.java

```diff
@@ -26,46 +25,37 @@
 import java.io.IOException;
 import java.nio.ByteBuffer;
 import java.nio.file.Path;
-import java.util.UUID;
 
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertNull;
-import static org.junit.jupiter.api.Assertions.assertThrows;
-import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.*;
 
 class DefaultCLIPropertiesTest {
 
-    @TempDir static Path pathToTmpDir;
+    @TempDir
+    static Path pathToTmpDir;
     private static String pathToPropertiesDir;
     private static String pathToEmptyProperties;
     private static String pathToOverrideProperties;
-    private static String pathToNoProperties;
+    @TempDir
+    static Path pathToNoProperties;
 
     @BeforeAll
     static void setUp() throws IOException {
-        final ClassLoader classLoader = DefaultCLIPropertiesTest.class.getClassLoader();
         System.setProperty("user.home", pathToTmpDir.toString());
         File mqttDir = new File(pathToTmpDir.toString() + "/.mqtt-cli");
         mqttDir.createNewFile();
-        pathToPropertiesDir = classLoader.getResource("PropertyFiles").getPath();
+        pathToPropertiesDir = DefaultCLIPropertiesTest.class.getResource("/PropertyFiles").getPath();
```

Comment:
Should it not be getClass().getResource()?

Reply:
we are in a static context here unfortunately